src/__test__/pages/experiments/[experimentId]/data-exploration/components/differential-expression-tool/DiffExprResults.test.jsx

```diff
@@ -214,4 +216,18 @@ describe('DiffExprResults', () => {
       lookupComponent.unmount();
     });
   });
+  it('Show comparison settings button works.', () => {
+    const component = mount(
+      <Provider store={store}>
+        <DiffExprResults experimentId='1234' onGoBack={jest.fn()} width={100} height={200} />
+      </Provider>,
+    );
+    const button = component.find(Button).at(1);
+    expect(button.childAt(0).text()).toEqual('Show settings');
+    button.simulate('click');
+    expect(button.childAt(0).text()).toEqual('Hide settings');
```

Comment:
similarly to my lower comment about this test, see if you can use scc selectors for finding the correct button and checking the text. See the existing code for specific examples.